front/src/Employees/Employee/Employee.js

```diff
@@ -1,7 +1,7 @@
 import * as React from 'react';
 import './Employee.css';
 
-const employee = ( props ) => {
+const employee = (props) => {
```

Comment:
Minor: let's name components starting with capital letter, e.g `Employee`

Reply:
> Minor: let's name components starting with capital letter, e.g `Employee`

@muchas I think we could introduce `eslint` to the project and use Airbnb style. I'll help @WKD622 with this to set it up correctly in PyCharm as well, this will improve code style and avoid comments like this one from you.